src/cleanlab_tlm/utils/rag.py

```diff
@@ -657,7 +657,10 @@ def __repr__(self) -> str:
     },
     {
         "name": "query_ease",
-        "criteria": "Determine whether the above User Request appears simple and straightforward. A bad User Request will appear either: disgruntled, complex, purposefully tricky, abnormal, or vague, perhaps missing vital information needed to answer properly. The simpler the User Request appears, the better. If you believe an AI Assistant could correctly answer this User Request, it is considered good. If the User Request is non-propositional language, it is also considered good.",
+        "criteria": """Determine whether the above User Request appears simple and straightforward.
+A bad User Request will appear either: ambiguous in intent, disgruntled, complex, purposefully tricky or abnormal.
+A good User Request is phrased clearly and expresses an achievable intent, even if it contains pronouns or references to prior context.
+Vagueness alone does not make a Request bad - since an AI assistant responsible for addressing the query could potentially access additional user context, be aware of domain-specific terms or access domain-specific knowledge bases to provide a proper response, so a User Request that seems to be missing information could still be good.""",
```

Comment:
I feel like the "an AI assistant" phrase came out of no where (like why is there suddenly an AI assistant involved).

I wonder if we should add

```
Determine whether the above User Request appears simple and straightforward for an AI assistant to handle.
```

or something similar in the first line

Reply:
I'm concerned you got rid of the statement which expresses what this Eval is supposed to be all about:

> If you believe an AI Assistant could correctly answer this User Request, it is considered good.

so would be good if you can point to evidence for why that is not good to include, or something like:

> If you believe an AI Assistant could properly handle this User Request, it is considered good.

And please ensure you confirmed the query_ease score is never low for basic conversational statements (when used appropriately within a dialogue) like:

- hi, hello, hey, good morning, bye, ...
- thank you, thanks, you're welcome, great!, ...

Reply:
Including that kind of phrase does help, but marginally. I've added it back (with a slight modification).

The basic conversational statements receive top scores.